src/SlackMessage.js

```diff
@@ -21,37 +22,35 @@ class SlackMessageRoot {
 }
 
 class SlackMessage extends SlackMessageRoot {
-  #_token = "";
   #_channel = "";
   #_ts = "";
+  #_slackGateway;
 
-  constructor(token, message = { channel: "", ts: "" }) {
+  constructor(slackToken, message = { channel: "", ts: "" }) {
     super();
-    if (!token) throw new Error("token is required");
-    this.#_token = token;
+    this.#_slackGateway = new SlackGateway(slackToken);
     this.#_channel = message.channel;
     this.#_ts = message.ts;
   }
 
   async _postRequest(method, body) {
     const response = await got.post(`https://slack.com/api/${method}`, {
       headers: {
-        Authorization: `Bearer ${this.#_token}`,
+        Authorization: `Bearer ${this.#_slackGateway.getToken()}`,
```

Comment:
This entire `post` call will be replaced, but for now just use the SlackGateway instance as the single source of truth for the token value.